src/blockchain/database/Filters.cpp

```diff
@@ -117,17 +111,12 @@ auto Filters::import_genesis(const blockchain::Type chain) const noexcept
         auto success{false};
 
         if (needHeader) {
-            auto headers = Vector<CFHeaderParams>{
-                {blockHash,
-                 [&](const auto& hex) {
-                     auto out = cfilter::Header{};
-                     const auto rc = out.DecodeHex(hex);
 
-                     OT_ASSERT(rc);
+            cfilter::Header header{};
+            OT_ASSERT(header.DecodeHex(genesis.first));
```

Comment:
Let's not do that. If one day someone puts a conditional compilation directive disabling OT_ASSERT in production, the statement will be silently lost.